src/components/automated/Conversation.vue

```diff
@@ -1,32 +1,47 @@
 <template>
-    <div class="conversation-container">
-        <chain-node v-model="chain"></chain-node>
-        <div>
-            <span @click="handelSave">Save</span>
-        </div>
+  <div class="conversation-container">
+    <chain-node v-model="chain"></chain-node>
+    <div>
+      <span @click="handelSave">Save</span>
     </div>
+  </div>
 </template>
 
 <script>
-    import tree from './models'
-    import ChainNode from './ChainNode'
-    export default {
-        name: 'conversation',
-        components: {
-            'chain-node': ChainNode,
-        },
-        data() {
-            return {
-                chain: tree,
-            }
-        },
-        methods: {
-            handelSave() {
-                console.log("chain:",JSON.stringify(this.chain));
-            }
-        }
-    }
+import tree from "./models";
+import ChainNode from "./ChainNode";
+export default {
+  name: "conversation",
+  components: {
+    "chain-node": ChainNode,
+  },
+  data() {
+    return {
+      chain: tree,
+    };
+  },
+  methods: {
+    handelSave() {
+      localStorage.setItem("chain", JSON.stringify(this.chain));
+      console.log("chain:", JSON.stringify(this.chain));
+    },
+    hanedelLoadChain() {
```

Comment:
functions name should be or like:
- `loadTreeFromLocalStorage()`
- `saveTreeToLocalStorage()`